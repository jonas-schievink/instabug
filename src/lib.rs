#[cfg(test)]
mod tests {
    use insta::assert_snapshot_matches;

    #[test]
    fn bla() {
        assert_snapshot_matches!(
            "bla",
            r#"
First line
Second line
Third line
"#
        );
    }
}
