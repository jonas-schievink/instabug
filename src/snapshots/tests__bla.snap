---
created: "2019-02-08T10:24:32.181491971Z"
creator: insta@0.6.1
source: src/lib.rs
expression: "r#\"\nFirst line\nSecond line\n\"#"
---

First line
Second line

